app.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Nango",
+    "description": "Get OAuth tokens for APIs - fast & secure",
+    "website": "https://www.namgo.dev",
+    "repository": "https://github.com/NangoHQ/nango",
+    "logo": "https://raw.githubusercontent.com/NangoHQ/nango/master/docs/static/img/nango-favicon.svg",
+    "stack": "container",
+    "image": "nango.docker.scarf.sh/nangohq/nango-server",
+    "env": {
+        "NANGO_SECRET_KEY": {
+            "generator": "secret",
+            "description": "Unique secret to be used for making API calls"
+        },
+        "NANGO_INSTALL_ID": {
+            "generator": "secret",
+            "description": "Unique install ID"
+        },
+        "SITE_URL": {
+            "description": "Use https://<your-app-name>.herokuapp.com replacing <your-app-name> with the value above. Or use a custom domain name if you plan to add one to the app later."
+        }
```

Comment:
There are a couple env variables that are missing for this to work properly.

The server needs access to the DB credentials to be able to read/write to it. On Render, there was a way to directly hook up the DB credentials to the render.yaml file via variables (because the database does not exist when the file is written obviously).

The missing env variables related to the DB are: 
NANGO_DB_USER
NANGO_DB_PASSWORD
NANGO_DB_HOST
NANGO_DB_PORT
NANGO_DB_NAME

We should also have this one with a fixed value:
NANGO_DB_SSL=true

There are 2 missing env variables related to the server instance: 
SERVER_PORT (Render listened to https requests on 443, it's probably the same for Heroku)
NANGO_SERVER_URL (we didn't have this one in the render.yaml because the code uses directly the RENDER_EXTERNAL_URL env variable, but we need it for Heroku)